notes/application-received.vbs

```diff
@@ -278,12 +314,21 @@ function get_caseload_array_by_type(caseload_type, available_caseload_array)
 	ReDim temp_array(0)
 	counter = 0
 
-	for i = 0 to UBound(all)
-		If all(i) = caseload_type Then
-			ReDim preserve temp_array(counter)
-			temp_array(counter) = things(i)
-			counter = counter + 1
-		End If
+	for i = 0 to UBound(all) - 1
+        If random_team_needed = TRUE Then  'This will be used to randomly select a PM team for the case to be transferred to for cash/snap pending caseload
```

Comment:
Add a backup to set random team needed to true if the caseload type has a question mark at the end.